common/utils.py

```diff
@@ -27,17 +27,12 @@
 
 
 class Utils:
-    def __init__(self):
-        self.msg_type = {
-            0: 'info:: ',
-            1: 'err:: '
-        }
 
     def report_slack(self, type, slack_msg, SLACK_HOOK):
         url = SLACK_HOOK['hostname'] + SLACK_HOOK['path']
-        prefix = self.msg_type.get(type, "")
-        payload = {"username": "webhookbot", "text": prefix + slack_msg, "icon_emoji": ":ghost:"}
-        resp = requests.post(url=url, data=json.dumps(payload))
+        payload = {"username": "webhookbot", "text": slack_msg, "icon_emoji": ":ghost:"}
```

Comment:
Report slack and send a slack notification are two methods for the sending alerts. We should remove one.

Reply:
i have reverted few changes. as one function will be used to report specific channel